src/app/drive/components/FileViewer/FileViewer.tsx

```diff
@@ -53,65 +55,119 @@ const FileViewer = ({ file, onClose, onDownload, downloader, show }: FileViewerP
   const isTeam = useAppSelector(sessionSelectors.isTeam);
   const userEmail: string = localStorageService.getUser()?.email || '';
 
+  const handleFileThumbnail = async (file: DriveFileData, fileBlob: Blob) => {
```

Comment:
[File implements Blob](https://developer.mozilla.org/en-US/docs/Web/API/File), therefore you should be able to do the implicit coercion by typing the `fileBlob` variable as `File`. Also, as a suggestion, you can name it `file` then, instead of `fileBlob`